### ISSUE TITLE
Corrupted fonts & images

### ISSUE THREAD

**Issue by qgthree** [Akryum/vue-cli-plugin-apollo#9]
Using "vue add apollo" on an existing project with images & fonts led to those files being corrupted. This includes files in both the public and src folders. I noticed those files are included when the install tells you "the following files have been updated / added."

**Comment by Akryum** [comment]
This is strange, do you have a reproduction repository?
Anyway, you should be able to recert the changes on those impacted files.

**Comment by qgthree** [comment]
Yeah, here ya go. This is before adding with "vue add apollo."[https://github.com/qgthree/vue-locust-apollo3](https://github.com/qgthree/vue-locust-apollo3). I reverted the changes on those files with no problem, but just wanted to report the issue.

**Comment by archSeer** [comment]
Same result here, it tampered with a bunch of logo and favicon files:

```
        modified:   public/android-chrome-192x192.png
        modified:   public/android-chrome-256x256.png
        modified:   public/apple-touch-icon.png
        modified:   public/favicon-16x16.png
        modified:   public/favicon-32x32.png
        modified:   public/favicon.ico
        modified:   public/mstile-150x150.png
        modified:   src/assets/logo.png
```

**Comment by dannyrb** [comment]
FWIW, I had this happen to me when invoking a different plugin on an existing project.

**Comment by squaretone** [comment]
vue-cli has yet to release the fix for this. Is there any temporary work-a-round that I can employ until they do?

**Comment by Akryum** [comment]
Revert the changes on those files using git.

Le jeu. 5 avr. 2018 à 16:12, Eric Lawrence <notifications@github.com> a
écrit :
